### DISCUSSION TITLE
Recent update has been causing Agent Ransack to crash

### DISCUSSION THREAD

**Discussion by melstephens** [valinet/ExplorerPatcher#2596] [General]
At some point in the last couple of weeks I've started to experience frequent crashes of Agent Ransack, a search enhancement tool. Presumably a clash triggered by an update of ExplorerPatcher, as previously both had co-existed happily for some time.

Windows 11 22H2 and 23H2
Explorer Patcher 22621.2861.62.1 and the previous release at least.

Crashes were frequent but not consistent, and occurred at the opening dialogue of Agent Ransack, prior to a search being performed. I've seen no other issues with any other software or the OS.

The only info I have from the authors of Agent Ransack is as follows -

Thanks for the error report. The error is being caused by a 3rd party program you have installed:
MODULE_NAME: ExplorerPatcher_amd64

IMAGE_NAME: ExplorerPatcher.amd64.dll

STACK_COMMAND: ~28s ; .ecxr ; kb

FAILURE_BUCKET_ID: INVALID_POINTER_WRITE_c0000005_ExplorerPatcher.amd64.dll!Unknown

**Comment by pyrates999** [comment]
Please check using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.

Then when the crash happens, copy and paste the results from there here.

**Reply by melstephens** [reply]
Apologies, I should have thought of that, having noticed before that there's a debug console. In the meantime I'd reset all of the ExplorerPatcher options back to defaults, and the issue with Agent Ransack disappeared. I then configured all of the ExplorerPatcher options again.... and still the issue has not reappeared. So, unfortunately I cannot at the moment pinpoint where the clash was. I'll bookmark this thread though, and update it in the event that I see it happening again.
